tests/test_api.py

```diff
@@ -446,45 +446,72 @@ def test_preprocess_error(self):
         processor = api.GradeCSVProcessor(course_id=self.course_id)
         row = {
             'block_id': self.usage_key,
+            'new_override-123402db': '1',
             'new_override-85bb02db': '-1',
             'user_id': self.learner.id,
             'csum': '07ec',
             'Previous Points': '',
         }
-        with self.assertRaisesMessage(ValidationError, 'Grade must be positive'):
+        with self.assertRaisesMessage(ValueError, 'Grade must be positive'):
```

Comment:
Should this be ValidationError?

Reply:
It was not meant for this one. I was trying to put `ValueError` on `not a number` error.